### ISSUE TITLE
Change links to Handling Author Errors section to be on relevant attributes

### ISSUE THREAD

**Issue by jnurthen** [w3c/core-aam#77]
The table at https://w3c.github.io/core-aam/#not_mapped includes the following incorrect information:

Handling Author Errors for States and Properties is referenced from the following attributes:

- aria-valuemax
- aria-valuemin
- aria-valuetext
- aria-valuenow (this one is correct)

It should be referenced from the following attributes:

- aria-checked
- aria-controls
- aria-expanded
- aria-level
- aria-valuenow

Note - I'm unsure what we should do about the implicit values for roles section (for example alert setting aria-live and aria-atomic) as this seems to be missing in core-aam unless I am not seeing it.
